src/common/ceph_context.cc

```diff
@@ -345,10 +345,10 @@ void CephContext::do_command(std::string command, cmdmap_t& cmdmap,
     if (!cmd_getval(this, cmdmap, "var", var)) {
       f->dump_string("error", "syntax error: 'perf reset <var>'");
     } else {
-     if(!_perf_counters_collection->reset(var))
+      if(!_perf_counters_collection->reset(var))
```

Comment:
> Should we not touch/fix these kind of cases?

please don't, unless you happen to be on it. otherwise, it would be a pain to `git blame`.